lib/elementSearch.js

```diff
@@ -31,52 +41,65 @@ function match(text, options = {}, ...args) {
                   : NodeFilter.FILTER_REJECT;
               },
             };
+
       const iterator = document.createNodeIterator(
         selectorElement,
         NodeFilter.SHOW_ALL,
         nodeFilter,
       );
+
       const exactMatches = [],
         containsMatches = [];
 
+      function checkIfRegexMatch(text, exactMatch) {
```

Comment:
I am finding it hard to understand this method especially because this works on a variable `searchText` which is outside it's scope. I recommend passing it as an input parameter.